modules/ROOT/pages/logging-and-debugging.adoc

```diff
@@ -7,6 +7,8 @@ MuleSoft provides a variety of logging and debugging options for Mule apps.
 
 Application logs and a runtime log help you monitor and troubleshoot your apps and the Mule server. For details, see xref:logging-in-mule.adoc[Configuring Logging].
 
+You can xref:enable-verbose-logging.adoc[Enable Verbose Logging] for Connectors and Modules to record debug level information in the logs. 
```

Comment:
...enable verbose logging for connectors and modules to record debug-level information....